syserol/dataImport.py

```diff
@@ -132,3 +132,10 @@ def createCube():
     assert ~np.any(np.all(np.isnan(cube), axis=(1, 2)))
 
     return cube, glyCube
+
+def funcRegImport():
```

Comment:
What does this function do? Are you using it? Delete if not.

Reply:
Yeah I think this was Robby's function but I haven't been using it so I will delete.